controlplane/controllers/ck8scontrolplane_controller.go

```diff
@@ -355,6 +352,11 @@ func (r *CK8sControlPlaneReconciler) updateStatus(ctx context.Context, kcp *cont
 	kcp.Status.ReadyReplicas = 0
 	kcp.Status.UnavailableReplicas = replicas
 
+	lowestVersion := ownedMachines.LowestVersion()
```

Comment:
Do we know how this `LowestVersion` is populated, is it derived from the actual node itself vs the `version` field in the cluster CRD? IIRC you can technically skip populating the version and use the snap revision for install.

Reply:
It's actually retrieving it from the ``Machine`` spec, which I think is supposed to be an actual k8s version:

https://github.com/kubernetes-sigs/cluster-api/blob/8523bb29cf5a76a592c208c8b28fad758621fb0b/util/collections/machine_collection.go#L273-L282